.github/workflows/test.yml

```
@@ -73,13 +76,16 @@ jobs:
             echo "::warning title=yt-dlp failed when testing script,file=plugin/yt_dlp_plugins/extractor/getpot_bgutil_script.py::yt-dlp returned $ret exit status"
           fi
 
-          if [[ "$script_response" != *"BgUtilScript: Generating POT via script: "* ]]; then
+          if [[ "$script_response" != *"[youtube] [pot:bgutil:script] Generating POT via script: "* ]]; then
             echo "::error title=BgUtilScript was not invoked,file=plugin/yt_dlp_plugins/extractor/getpot_bgutil_script.py::\
           BgUtilScript was not invoked"
             [ "$exit_code" -eq 0 ] && exit_code=1
           fi
 
-          pot_response=$(echo "$script_response" | grep -A1 -m1 '^\[debug\]\ \[GetPOT\]\ BgUtilScript:\ stdout:$' | tail -n1 | jq -C '.poToken')
+          echo "$script_response"
+          pot_response=$(echo "$script_response" | grep -A1 -m1 '\[youtube\] \[pot:bgutil:script\] TRACE: stdout:' | tail -n1 | jq -r '.poToken')
+
+
```

```suggestion
          pot_response=$(echo "$script_response" | grep -A1 -m1 '\[youtube\] \[pot:bgutil:script\] TRACE: stdout:' | tail -n1 | jq -r '.poToken')
```
remove the redundant echo
theres already a `echo` up there